### ISSUE TITLE
Pasting is broken

### ISSUE THREAD

**Issue by magthe** [syl20bnr/spacemacs#8672]
#### Description :octocat:
Pasting is broken and it seems related to 58458f2d2abcc1211444c2060ab598f55e518da4

#### Reproduction guide :beetle:
- Start Emacs
- Try to paste something with `P`

*Observed behaviour:* :eyes: :broken_heart:
Nothing is pasted and a message "Symbol's function definition is void: evil-mc-get-cursor-count" is displayed.

*Expected behaviour:* :heart: :smile:
That the text is pasted.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.1.1
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 5cb0fd73)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-help-tooltip t auto-completion-enable-snippets-in-popup t auto-completion-enable-sort-by-usage t)
 dash docker evil-commentary evil-snipe git gtags helm mu4e syntax-checking vim-powerline
 (c-c   :variables c-c  -enable-clang-support t c-c  -enable-irony-support t)
 clojure emacs-lisp python
 (haskell :variables haskell-completion-backend 'intero)
 html latex markdown org pandoc yaml archlinux clipboard-fix)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11

**Comment by kain88-de** [comment]
As a workaround I get pasting to work again by activating evil-mc-mode

**Comment by sebhahn** [comment]
experience same problem

**Comment by hallabro** [comment]
According to as `git bisect` commit `58458f2d2abcc1211444c2060ab598f55e518da4` is the culprit. You can verify this by `git checkout 58458f2d2abcc1211444c2060ab598f55e518da4~1`.

**Comment by et2010** [comment]
I got the same problem.

**Comment by syl20bnr** [comment]
Thanks for the report, should be fixed in develop.

**Comment by mahinshaw** [comment]
This fixed the issue for me. Thanks!

**Comment by nixmaniack** [comment]
Still getting error with the latest fix:

```
Debugger entered--Lisp error: (invalid-function (lambda (&optional count &optional register) "Disable paste transient state if there is more that 1 cursor." (interactive "p") (if (spacemacs//paste-transient-state-p) (spacemacs/paste-transient-state/evil-paste-after) (evil-paste-after count register))))
  spacemacs/evil-mc-paste-after(1)
  funcall-interactively(spacemacs/evil-mc-paste-after 1)
  call-interactively(spacemacs/evil-mc-paste-after nil nil)
  command-execute(spacemacs/evil-mc-paste-after)
```

**Comment by madand** [comment]
#8692 fixes the error mentioned by @nixmaniack.

**Comment by syl20bnr** [comment]
Thanks @madand

**Comment by korayal** [comment]
Would this be about the change in evil-mc? because initially the function name was `spacemacs-evil/evil-mc-paste-{after,before}` and these functions were needed to be added to the list of known commands defined in the evil-mc project. 

Then the name of these functions were changed to `spacemacs/evil-mc-paste-{after,before}`, so I made another PR to evil-mc accordingly.

So people that are using an older version of evil-mc will probably get the error @nixmaniack mentioned.
